src/controllers/who.is.making.filing.controller.ts

```diff
@@ -12,9 +12,16 @@ import {
   OVERSEAS_ENTITY_DUE_DILIGENCE_WITH_PARAMS_URL
 } from "../config";
 import { getWhoIsFiling, postWhoIsFiling } from "../utils/who.is.making.filing";
+import { getApplicationData } from "../utils/application.data";
+import * as config from "../config";
+import {OverseasEntityKey, Transactionkey} from "../model/data.types.model";
```

Comment:
can you run the linter (it should auto run via husky when you git commit), the linter should highlight that spaces needed either side of the {} in teh import.

Reply:
All fixed now. Will be in next push.